### PR TITLE
Release notes for 5.5.57-38.9

### PR DESCRIPTION
**Pull request by azhebel**

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.5.57-38.9.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.. rn:: 5.5.57-38.9
+
+==========================
+Percona Server 5.5.57-38.9
+==========================
+
+Percona is glad to announce the release of Percona Server 5.5.57-38.9
+on August 17, 2017.
+Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.55-38.8/>`_
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-5.5.57-38.9.rst:10]: Wrong URL, should be 5.5.57-38.9

**Comment by azhebel** [comment]
Based this on wrong commit

File: doc/source/release-notes/Percona-Server-5.5.57-38.9.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Bugs Fixed
+==========
+
+* :bug:`1661488`: Allowed to enable :variable:`userstat`
+  for existing connections
+  and keep tracking of concurrent connections
+  after running ``FLUSH CLIENT_STATISTICS`` or ``FLUSH USER_STATISTICS``.
+
+* :bug:`1673656`: Added support of Subject Alternative Names (SAN)
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-5.5.57-38.9.rst:40]: Please add more details regarding lp:1673656 from 
https://raw.githubusercontent.com/percona/percona-server/5.6/doc/source/release-notes/Percona-Server-5.6.36-82.1.rst
Including Compatibility matrix

File: doc/source/release-notes/Percona-Server-5.5.57-38.9.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+in the `5.5.57-38.9 milestone on Launchpad
+<https://launchpad.net/percona-server/+milestone/5.5.57-38.9>`_.
+
+.. note:: Red Hat Enterprise Linux 5 (including CentOS 5 and other derivatives),
+   Ubuntu 12.04 and older versions are no longer supported by Percona software.
+
+New Features
+============
+
+* Added support and packages for Debian 9 (stretch).
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-5.5.57-38.9.rst:28]: Support of OpenSSL 1.1 must also be highlighted
`|Percona Server| can now be built with support of OpenSSL 1.1.`

Comment (laurynas-biveinis) [reply doc/source/release-notes/Percona-Server-5.5.57-38.9.rst:28]: This and almost all bugfix descriptions should be copied over from the last 5.6/5.7 release notes and edited if applicable for any 5.5 specifics

**Review by laurynas-biveinis — COMMENTED** [review]
Needs upstream bug list update.
Please don't close the PR if you are about to submit it again, just push a corrected branch with the same name, to keep the review discussion in a single PR

**Pull request closed without merging**